### ISSUE TITLE
Accessibility: the logo shouldn't be made in two parts

### ISSUE THREAD

**Issue by ghost** [opentofu/registry-ui#148]
Currently, the word "registry" is not included in the link on the header and the `aria-label` for the SVG image is "Home". Instead, the SVG image and the "registry" word should be part of the link and the aria label should read "OpenTofu Registry".
